deploy-manage/deploy/cloud-on-k8s/configuration-examples-standalone.md

```diff
@@ -47,9 +47,23 @@ Deploys two {{es}} clusters and two {{kib}} instances together with single Elast
 
 ## Storing local state in host path volume [k8s_storing_local_state_in_host_path_volume]
 
-{{agent}} managed by ECK stores local state in a host path volume by default. This ensures that {{integrations}} run by the agent can continue their work without duplicating work that has already been done after the Pod has been recreated for example because of a Pod configuration change. Multiple replicas of an agent, for example {{fleet}} Servers, can not be deployed on the same underlying {{k8s}} node as they would try to use the same host path. There are 2 options for managing this feature:
+{{agent}} managed by ECK stores local state in a host path volume by default. This ensures that {{integrations}} run by the agent can continue their work without duplicating work that has already been done after the Pod has been recreated for example because of a Pod configuration change. Multiple replicas of an agent, for example {{fleet}} Servers, can not be deployed on the same underlying {{k8s}} node as they would try to use the same host path. There are 2 options for managing this feature that differ depending on the version of Elastic Agent.
+
+
+::::{tab-set}
+
+:::{tab-item} {{agent}} 8.16 and higher
+
+1. If local state storage in `hostPath` volumes is not desired this can be turned off by configuring an `emptyDir` volume instead.
+2. Run {{agent}} with a `hostPath` volume. No additional configuration is required.
```

Comment:
These are not sequential steps, so you should not use a numbered list

```suggestion
* If local state storage in `hostPath` volumes is not desired this can be turned off by configuring an `emptyDir` volume instead.
* Run {{agent}} with a `hostPath` volume. No additional configuration is required.
```